pkg/controller/installation/core_controller.go

```diff
@@ -455,7 +455,7 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 
 	var managerTLSSecret *corev1.Secret
 	managerTLSSecret, err = utils.ValidateCertPair(r.client,
-		render.ManagerTLSSecretName,
+		render.VoltronTLSSecretName,
```

Comment:
Ignore this. I'm not sure why I added this change. This should stay `ManagerTLSSecretName`.